### DISCUSSION TITLE
Why are the Proton builds lacking components?

### DISCUSSION THREAD

**Discussion by trymeouteh** [Kron4ek/Wine-Builds#70] [Q&A]
Why are the Proton builds lacking the python scripts, builtin dlls, DXVK and vkd3d-proton? Why not include the python scripts, dlls and install the same versions of DXVK & vkd3d-proton that is used in that version of Proton.

For example in Proton 7.0-3, it uses DXVK v1.10.1 and vkd3d-proton v2.6 and dxvk-nvapi v0.5.3.

**Comment by Kron4ek** [comment]
I compile only the [wine part](https://github.com/ValveSoftware/wine) from the Proton project, which does not include any of that stuff. Of course i could add these files manually, but i just see no reason.

**Reply by trymeouteh** [reply]
How do you add the python scripts and other builtin dlls? I understand how to install DXVK and I am sure vkd3d-proton installation is similar.

**Reply by Kron4ek** [reply]
The python script is [here](https://github.com/ValveSoftware/Proton/blob/proton_7.0/proton).
